packages/react-vapor/src/components/itemBox/ItemBox.tsx

```diff
@@ -20,6 +20,7 @@ export interface IItemBoxProps {
     append?: IContentProps;
     onOptionClick?: (option?: IItemBoxProps, index?: number) => void;
     selectedDisplayValue?: string;
+    hoverTooltipValue?: string;

```

Comment:
- Our tooltips always appear on hover so I would drop that part from the prop name.
- This prop will intrinsically be linked to a selected item so I would specify that in the name.
- Our other tooltip prop is of type `ITooltipProps`, I think this one should be the same.

```suggestion
    selectedTooltip: ITooltipProps;
```